test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -614,17 +611,26 @@ def test_cgroup_periodic_update(self):
         self.server.status(JOB, [ATTR_o, ATTR_e], jid)
         o = j.attributes[ATTR_o]
         self.mom.log_match("%s;update_job_usage: " % jid +
+                           "CPU usage: 0.000 secs",
+                           max_attempts=2,
+                           starttime=self.server.ctime)
+        self.mom.log_match("%s;update_job_usage: " % jid +
+                           "Memory usage: mem=0b",
```

Comment:
Isn't update_job_usage method converting mem, vmem to kb?

Reply:
It does, but the log message shows bytes because it's smaller than 1kB.